### PR TITLE
Enhance the Clamor README

### PR DESCRIPTION
**Pull request by satendrapalsingh**
Add standard README sections like Introduction, Installation, Usage, License, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,69 @@
...
+The App Explorer will now connect with your local node and will show the blocks being produced by your node in real-time.
+
+### Connecting to your front-end applicaton
+
+You can also create your own front-end application to connect with and consume your Clamor node backend.
+
+For an example of such an application, see [this](https://github.com/fragcolor-xyz/clamor-frontend). 
+
+## License
+Clamor source code is licensed under the [The Unlicense](./LICENSE).
```
Comment (sinkingsugar) [inline README.md]: Actually it's complex.
You should mention our pallets are BUSL-1.0.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,69 @@
...
+1. Run you Clamor node locally
+2. Head over to Polkadot.js' [App Explorer](https://polkadot.js.org/apps/#/explorer)
+3. Click the top-left Pokadot icon on the header of the page
+4. Expand the **Development** sub-menu (at the bottom of the list)
+5. Click **Local Node** to enable it
+6. Click **Switch** at the top of the panel
+
+The App Explorer will now connect with your local node and will show the blocks being produced by your node in real-time.
+
+### Connecting to your front-end applicaton
```
Comment (sinkingsugar) [inline README.md]: wrong, we need to delete clamor-frontend
dotapps is the way.

File: README.md (hunk 1 above)
Comment (satendrapalsingh) [reply README.md]: BUSL-1.0 seems to be [deprecated](https://mariadb.com/bsl-faq-adopting/#:~:text=BSL%20version%201.0%20is%20now%20deprecated) and the official link to the license is only for [BUSL-1.1](https://spdx.org/licenses/BUSL-1.1.html) now. No link for BUSL-1.0.

Should I mention BUSL-1.0 and link to the BUSL-1.1 instead? or just mention BUSL-1.0 without any link?
*I'm assuming we are not immediately updating the pallets license to BUSL-1.1

File: README.md (hunk 2 above)
Comment (satendrapalsingh) [reply README.md]: Polkadot's [dotapps](https://github.com/polkadot-js/dotapps.io)?

File: README.md (hunk 1 above)
Comment (satendrapalsingh) [reply README.md]: Just checked, the pallets use BUSL-1.1 license. So will update README accordingly.

Comment (satendrapalsingh) [reply README.md]: Pallets license info added to README.

File: README.md (hunk 2 above)
Comment (satendrapalsingh) [reply README.md]: Removed the section about connecting own front-end application to Clamor node.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,63 @@
...
-```
-
-## Populate test assets via docker/cbl
-```sh
-docker run --rm --user root --network host -v `pwd`:/data chainblocks/cbl cbl /data/chains/add-test-assets.edn
-```
\ No newline at end of file
+# Clamor
+
+[![license](https://img.shields.io/github/license/fragcolor-xyz/clamor)](./LICENSE)
```
Comment (sinkingsugar) [inline README.md]: Remove this

File: README.md [hunk 4]
```diff
@@ -1,16 +1,63 @@
...
+2. Head over to the [App Explorer](https://polkadot.js.org/apps/#/explorer)
+3. Click the top-left Pokadot icon on the header of the page
+4. Expand the **Development** sub-menu (at the bottom of the list)
+5. Click **Local Node** to enable it
+6. Click **Switch** at the top of the panel
+
+The App Explorer will now connect with your local node and will show the blocks being produced by your node in real-time. 
+
+## License
+Pallets are licensed under the [BUSL-1.1](https://spdx.org/licenses/BUSL-1.1.html) license, while the rest of the Clamor source code is licensed under [The Unlicense](./LICENSE) license.
```
Comment (sinkingsugar) [inline README.md]: I would probably go full BUSL

File: README.md (hunk 3 above)
Comment (satendrapalsingh) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (satendrapalsingh) [reply README.md]: updated

**Review by Kryptos-FR — APPROVED** [review]

**Pull request merged**